fhir-term/src/main/java/com/ibm/fhir/term/util/CodeSystemSupport.java

```diff
@@ -393,18 +425,6 @@ public static Element getConceptPropertyValue(Concept concept, Code code) {
         return DESCENDANTS_AND_SELF_CACHE.computeIfAbsent(code.getValue(), k -> computeDescendantsAndSelf(codeSystem, code));
     }
 
-    /**
-     * Get the appropriate string mapper for the given code system based on its case sensitivity.
-     *
-     * @param codeSystem
-     *     the code system
-     * @return
-     *     the appropriate string mapper for the given code system based on its case sensitivity
-     */
-    public static Function<Concept, java.lang.String> getStringMapper(CodeSystem codeSystem) {
-        return isCaseSensitive(codeSystem) ? STRING_MAPPER : NORMALIZED_STRING_MAPPER;
-    }
```

Comment:
this seemed useful to me...we don't need it any more?

Reply:
oh, now I see it just got moved up to line 273:  getCodeValueFunction(CodeSystem codeSystem)